functions/package.json

```diff
@@ -8,28 +8,42 @@
     "shell": "npm run build && firebase functions:shell",
     "start": "npm run shell",
     "deploy": "firebase deploy --only functions",
-    "logs": "firebase functions:log"
+    "logs": "firebase functions:log",
+    "dev": "concurrently \"nodemon\" \"nodemon -x tsoa spec-and-routes\"",
+    "build:routes": "tsoa spec-and-routes && tsc",
+    "start:server": "node build/src/server.js"
   },
   "engines": {
-    "node": "12"
+    "node": "14"
```

Comment:
do we have to move up from 12? would be nice to be consistent with course-scraper so we don't need to change node versions between developing?